recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f3ae4371f2a9cf493c77190010e73270ddc658904618937889ce14c0ecebb09
+
+build:
```

Comment:
Please, delete the ``build.sh`` and ``bld.bat``. After that please add in your ``build`` section
```
  script:
    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
```

Otherwise, this recipe cannot be ``noarch``

Reply:
@marcelotrevisani Thanks! I just pushed this change.